.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,22 +4,21 @@ v    Before smashing the submit button please review the checkboxes.
 v    If a checkbox is n/a - please still include it but + a little note why
 ☺ > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > >  -->
 
+closes: #XXXX
+
 ## Description
 
 <!-- Add a description of the changes that this PR introduces and the files that
 are the most critical to review.
 -->
 
-closes: #XXXX
```

Comment:
not blocking, but I personally like seeing what issue it closes on the bottom -- it's weird seeing it above a header.